src/server/managers/AppVideoConfProvider.ts

```diff
@@ -86,8 +87,12 @@ export class AppVideoConfProvider {
                 params: runContextArgs,
             });
 
-            return result as string;
+            return result as string | boolean | Array<IBlock> | undefined;
         } catch (e) {
+            if (method === AppMethod._VIDEOCONF_IS_CONFIGURED && e?.code === JSONRPC_METHOD_NOT_FOUND) {
+                return true;
+            }
+
```

Comment:
```suggestion
            if (e?.code === JSONRPC_METHOD_NOT_FOUND) {
                if (method === AppMethod._VIDEOCONF_IS_CONFIGURED) {
                    return true;
                }
                if (![AppMethod._VIDEOCONF_GENERATE_URL, AppMethod._VIDEOCONF_CUSTOMIZE_URL].includes(method)) {
                    return undefined;
                }
            }

```

Just to avoid logging "method not found" errors for methods that are optional.